test-framework/keycloak-server/src/main/java/io/quarkus/test/keycloak/client/KeycloakTestClient.java

```diff
@@ -16,7 +16,7 @@
 
 public class KeycloakTestClient implements DevServicesContext.ContextAware {
 
-    private final static String AUTH_SERVER_URL_PROP = "quarkus.oidc.auth-server-url";
+    private final static String AUTH_SERVER_URL_PROP = "client.quarkus.oidc.auth-server-url";
```

Comment:
It may not be quite right to remove `quarkus.oidc.auth-server-url`, as it won't work if custom test resource factories set `quarkus.oidc.auth-server-url` after launching a container.

Can you please update the code to fallback to `quarkus.oidc.auth-server-url` if the `client.quarkus.oidc.auth-server-url` is not available ?
Or may be it is better to just ignore the warnings you've mentioned ? If it works with the warnings then I'd leave `quarkus.oidc.auth-server-url`...

Reply:
Sure, I'll update the code